docs/topics/templates/document-attributes.adoc

```diff
@@ -13,19 +13,19 @@
 ifdef::mta[]
 :ProductName: Migration Toolkit for Applications
 :DocInfoProductName: Migration Toolkit for Applications
-:DocInfoProductNumber: 6.0
+:DocInfoProductNumber: 6.1
 :ProductShortName: MTA
 :LC_PSN: mta
 :DocInfoProductNameURL: migration_toolkit_for_applications
 :WebName: user interface
 :WebNameTitle: User Interface
 :WebNameURL: user_interface_guide
 :WebConsoleBookName: {WebNameTitle} Guide
-:ProductVersion: 6.0.1
+:ProductVersion: 6.1.0
 :PluginName: MTA plugin
-:MavenProductVersion: 6.0.1.GA-redhat-00006
-:ProductDistributionVersion: 6.0.1.GA-redhat
-:ProductDistribution: mta-6.0.1.GA-offline.zip
+:MavenProductVersion: 6.1.0.GA-redhat
```

Comment:
@sshveta - the link you sent did not look it gave the right Maven reference. 
Please can I double check as I need to be certain we have the correct references?

Reply:
@dymurray - please can i ask for your help on this and please can you confirm the following values:

:MavenProductVersion: 6.0.1.GA-redhat-00006
:ProductDistributionVersion: 6.0.1.GA-redhat
:ProductDistribution: mta-6.0.1.GA-offline.zip

Ta muchly

Reply:
Yes, looks good to me.

Reply:
To be clear, when I said "looks good to me", those are the 6.0.1 values. 

For 6.1.0 the changes in the PR looks sane. @sshveta if you can confirm the actual maven product reference that'd be great.

Reply:
@mrizzi may also know the answer to this.

Reply:
It's MTA-6.1.0.GA-maven
![Screenshot from 2023-04-21 08-43-56](https://user-images.githubusercontent.com/3866976/233678981-3fe65757-a2e9-47b7-a1be-8da50328049d.png)

Reply:
@anarnold97 the value is partial, the complete one is
```suggestion
:MavenProductVersion: 6.1.0.GA-redhat-00005
```